docs/index.bs

```diff
@@ -1188,55 +1188,58 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
         1. Return |promise|.
     </section>
 
-    <section algorithm="clients-getall">
-      <h4 id="clients-getall">{{Clients/matchAll(options)}}</h4>
+    <section algorithm="clients-matchall">
+      <h4 id="clients-matchall">{{Clients/matchAll(options)}}</h4>
 
       The <dfn method for="Clients"><code>matchAll(|options|)</code></dfn> method *must* run these steps:
 
-        1. Let |promise| be a new <a>promise</a>.
-        1. Run these substeps <a>in parallel</a>:
-            1. Let |targetClients| be an empty array.
-            1. For each [=/service worker client=] |client| whose [=service worker client/origin=] is the <a lt="same origin">same</a> as the associated [=ServiceWorkerGlobalScope/service worker=]'s [=environment settings object/origin=]:
+        1. Let |promise| be [=a new promise=].
+        1. Run the following steps [=in parallel=]:
+            1. Let |targetClients| be a new [=list=].
+            1. For each [=/service worker client=] |client| whose [=service worker client/origin=] is the [=same origin|same=] as the associated [=ServiceWorkerGlobalScope/service worker=]'s [=environment settings object/origin=]:
                 1. If |client| is an [=environment settings object=], then:
-                    1. If |client| is not a <a>secure context</a>, continue to the next iteration of the loop.
-                1. Else:
-                    1. If |client|’s <a>creation URL</a> is not a <a>potentially trustworthy URL</a>, continue to the next iteration of the loop.
-                1. If |options|.{{ClientQueryOptions/includeUncontrolled}} is false, then:
-                    1. If |client|'s <a>active service worker</a> is not the associated [=ServiceWorkerGlobalScope/service worker=], continue to the next iteration of the loop.
-                1. If |options|.{{ClientQueryOptions/includeReserved}} is false, then:
-                    1. If |client|'s [=environment/execution ready flag=] is unset, continue to the next iteration of the loop.
+                    1. If |client| is not a [=secure context=], [=continue=].
+                1. Otherwise:
```

Comment:
Nice cleanup! I think "Else" is better here for consistency though.

Reply:
Will change it for consistency, but what's the correct form here @annevk? https://infra.spec.whatwg.org/#variables uses "otherwise", but infra doesn't specifically cover if/else/otherwise.

Reply:
We should define conditionals more clearly I suppose, raise it? I suspect we'd go with "Otherwise" given the overwhelming precedent, but who knows.